src/httpserver.cpp

```diff
@@ -169,8 +169,7 @@ static bool InitHTTPAllowList()
     rpc_allow_subnets.push_back(CSubNet(localv4, 8));      // always allow IPv4 local subnet
     rpc_allow_subnets.push_back(CSubNet(localv6));         // always allow IPv6 localhost
     for (const std::string& strAllow : gArgs.GetArgs("-rpcallowip")) {
-        CSubNet subnet;
-        LookupSubNet(strAllow, subnet);
+        CSubNet subnet = LookupSubNet(strAllow);
```

Comment:
nit, here and elsewhere:

```suggestion
        const CSubNet subnet = LookupSubNet(strAllow);
```

Reply:
Done